mungegithub/mungers/submit-queue.go

```diff
@@ -1043,6 +1053,7 @@ func (sq *SubmitQueue) validForMergeExt(obj *github.MungeObject, checkStatus boo
 	mergeContexts := mungeopts.RequiredContexts.Merge
 	retestContexts := mungeopts.RequiredContexts.Retest
 	additionalLabels := sq.AdditionalRequiredLabels
+	claYesLabels := sq.ClaYesLabels
```

Comment:
Don't forget to use `claYesLabels` instead of `sq.ClaYesLabels` below. 😄

Reply:
Oh, what I was doing. Sorry about that. 
PTAL